packages/react-cards/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/react-cards",
-  "version": "1.0.0-beta.58",
+  "version": "9.0.0-alpha.0",
```

Comment:
is `cards` a converged component ? I have lost track of these discussions at this point sorry

Reply:
We put this into items to discuss in offline: bump only released things vs bump all packages that pretend to be converged.

Reply:
Reverted.